flow/envs/loop/lane_changing.py

```diff
@@ -83,8 +83,8 @@ def action_space(self):
         max_decel = self.env_params.additional_params["max_decel"]
         max_accel = self.env_params.additional_params["max_accel"]
 
-        lb = [-abs(max_decel), -1] * self.vehicles.num_rl_vehicles
-        ub = [max_accel, 1] * self.vehicles.num_rl_vehicles
+        lb = [-abs(max_decel), -1] * self.scenario.vehicles.num_rl_vehicles
```

Comment:
why does vehicles no longer have a reference to the number of rl vehicles? This seems like a break int he abstraction

Reply:
This is a minor bug that needs to be resolved. This is part of an already existing issue. Hoping to resolve once things calm down